Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "ssh-agent-lib"
 description = "A collection of types for writing custom SSH agents"
 version = "0.4.0"
-license = "MIT"
+license = "MIT OR Apache-2.0"
 authors = [
```

Comment:
I think we need to adjust this line too. We're *very* grateful to the original authors :bow: but currently it's @baloo that's maintaining it and @jcspencer who sends occasional PRs. :thinking: